sql/updates/latest-dev.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE _timescaledb_catalog.hypertable DROP CONSTRAINT hypertable_replication_factor_check, ADD CONSTRAINT hypertable_replication_factor_check CHECK (replication_factor > 0 OR replication_factor = -1);
```

Comment:
Why is this constraint needed? There is already unnamed constraint created.
And if it needed, can it be split to multiple lines for readability?

Reply:
This like alters the existing constraint, which is incorrect. The only way to "change" constraints is to drop and add it back again with the new definition.

Reply:
> This line alters the existing constraint, which is incorrect.

If this line is incorrect, why it is still kept. I guess you meant something else :)

The constraint is re-created during `_timescaledb_catalog.hypertable` re-creation after the drop. So I don't understand why constraint `hypertable_replication_factor_check` is needed. I guess I still miss something.

Reply:
It is still unclear why it is needed. See my comment above.

Reply:
The existing old constraint is incorrect. That's why it is recreated. I guess what you are saying is that this isn't needed because the update script for rc4 was already changed with this fix.

This change accounts for updates from RC4 installations to latest (2.0.0).